packages/frontend-api/src/Model/Mutation/Customer/DeliveryAddress/DeliveryAddressDataFactory.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Mutation\Customer\DeliveryAddress;
+
+use Overblog\GraphQLBundle\Definition\Argument;
+use Shopsys\FrameworkBundle\Model\Country\CountryFacade;
+use Shopsys\FrameworkBundle\Model\Customer\Customer;
+use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressData;
+use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressDataFactory as FrameworkProductRepository;
+use Shopsys\FrameworkBundle\Model\Pricing\Exception\InvalidArgumentException;
+
+class DeliveryAddressDataFactory
```

Comment:
yes! :ok_hand: :green_heart:

Reply:
maybe, and just maybe, it could have been named `DeliveryAddressDataApiFactory` so there are no conflicts in naming with the framework one :slightly_smiling_face:

Reply:
I liked that name much more. It helps to get rid off services ambiguous names :heart: 

Let's rewrite whole `fronted-api` package :pray:  :see_no_evil: